library/systemd/README.md

```diff
@@ -0,0 +1,76 @@
+# Systemd actions
+
+The following table describes the actions to perform over a systemd service and its associated socked (if any) when we try to start, stop, restart or reload the service.
+
+
+| Status | Start | Stop | Restart | Reload |
+|---|---|---|---|---|
+| Socket running | nothing | stop socket | stop socket and start service/socket (depending on start mode) | nothing |
+| Service running | nothing  | stop service  | stop service and start service/socket (depending on start mode) | reload service (if it support, otherwise restart) |
+| Socket & Service running | nothing | stop service and socket | stop service and start service/socket (depending on start mode) | reload service (if it support, otherwise restart) |
+| nothing runs | start service/socket (depending on start mode) | nothing | start service/socket (depending on start mode) | start service/socket (depending on start mode) |
+
+
+## Actions over a `SystemService`
+
+In systemd, it might happen that a service is compose by a set of units (services, sockets, paths and so on). This class `SystemService` is able to group those units and handle them together.
```

Comment:
This might confuse the reader in the direction of CompoundService, which is something else.
As I understand it, this class only handles the concept of Laziness: activating the real Service via a Socket or a Path, or a Timer, which is its light-weight stand-in/proxy. In case we want to add Paths and Timers (there are no other proxies for services*), we could use the word Proxy (or StandIn but that is a hyphenated-word, bother).

*: For systemd.mount there is a systemd.automount stand-in, but we do not handle mounts.

Reply:
Examples:
- ca-certificates.path that runs ca-certificates.service (Type=oneshot) when a cert file is added/removed.
- backup-rpmdb.timer (oneshot, backs it up daily)